helper/src/main/java/io/github/libxposed/helpers/BaseFinder.java

```diff
@@ -0,0 +1,254 @@
+package io.github.libxposed.helpers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.lang.reflect.AccessibleObject;
+import java.lang.reflect.Member;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@SuppressWarnings({"unchecked", "unused"})
+abstract class BaseFinder<T extends Member, S> {
```

Comment:
For extendibility, consider adding `includes` and `excludes` filter for parameter types and exception types.

Reply:
This one is not resolved.